package.json

```diff
@@ -10,8 +10,8 @@
     "start": "yarn ts-node node_modules/.bin/webpack serve",
     "start:console": "./scripts/start-console.sh",
     "start:console:oauth": "./scripts/start-console-with-oauth.sh",
-    "start:console:minikube": "./scripts/start-console-minikube.sh",
-    "start:console:minikube:configure": "source ./scripts/configure-minikube.sh && ./scripts/start-console-minikube.sh",
+    "start:console:k8s": "./scripts/start-console-k8s.sh",
+    "start:console:k8s:configure": "source ./scripts/configure-minikube.sh && ./scripts/start-console-k8s.sh",
```

Comment:
how about `configure:minikube` and `configure:kind` ?

Reply:
makes sense to me too, updated